api/promises-openapi.yml

```diff
@@ -266,3 +259,13 @@ components:
             type: string
         data:
           type: string
+
+    SearchPromisesResponseObj:
```

Comment:
Can we leave this inline?

Reply:
this is an openapi thing. u shouldnt do inline cause it results in anonymous structs making it harder to deserialize.

Reply:
Then we need to be consistent and change everywhere, the commands will need to be updated and we should change search schedules definition to match.

Reply:
yea np. in general let's avoid inline stuff. in my experience, it causes trouble down the line.